packages/test-exclude/index.js

```diff
@@ -177,10 +177,10 @@ exportFunc.defaultExclude = [
     'coverage/**',
     'packages/*/test/**',
     'test/**',
-    'test{,-*}.js',
-    '**/*{.,-}test.js',
+    'test{,-*}.{,c,m}js',
```

Comment:
For this (and the next line) I'd prefer that we write out each extension: `{js,cjs,mjs}`.  test-exclude is not a web module so the 4 character savings is not that big and I think this makes it much easier to read.

@bcoe Should we be adding `ts` to the list of excluded tests?

Reply:
>  I'd prefer that we write out each extension: `{js,cjs,mjs}`. 

5cf2388b74cc6d12232b4fd497bb49b04e07487e

Reply:
I spoke with @bcoe in chat, would you mind adding `ts` to the test file extensions?

Reply:
8cba44077f9a5135d50ef28b5d70f41b964f1067